src/Amazon.Lambda.Tools/LambdaPackager.cs

```diff
@@ -451,7 +451,12 @@ private static bool ValidateDependencies(IToolLogger logger, string targetFramew
         /// <param name="publishLocation"></param>
         private static void FlattenPowerShellRuntimeModules(IToolLogger logger, string publishLocation, string targetFramework)
         {
-            var runtimeModuleDirectory = Path.Combine(publishLocation, "runtimes/win/lib/netcoreapp3.1/Modules");
+            var runtimeModuleDirectory = targetFramework switch
```

Comment:
I don't think we need to have the switch. You can just do `runtimeModuleDirectory = $"runtimes/unix/lib/{targetFramework}/Modules"`. Then we don't have to go back and update this for .NET 8.

Reply:
Fixed